natero/spec/natero/event_spec.rb

```diff
@@ -0,0 +1,134 @@
+require 'spec_helper'
+
+describe Natero::Event do
+  let(:valid_message) { true }
+  let(:api_key) { '1000' }
+  let(:event_api_key) { '5000' }
+  let(:event_auth_key) { '12345' }
+  let(:event) { Struct.new(:id, :name) { def to_h; {id: id, name: name}; end } }
+
+  before do
+    # allow(Natero.configuration).to receive(:account_api_key).and_return(api_key)
+    allow(Natero.configuration).to receive(:event_auth_key).and_return(event_auth_key)
+    allow(Natero.configuration).to receive(:event_api_key).and_return(event_api_key)
+  end
+
+  describe 'create' do
+    it 'sends the proper request' do
+      evt = event.new(1, 'test')
+      request_url = "https://events.natero.com/v1/#{event_auth_key}/#{event_api_key}"
+      data = {:body=>{id:1, name:'test', action:'test'},
+              :headers=>{'Content-Type': 'application/json'}}
+      method = :post
+
+      expect(HTTParty).to receive(method).with(request_url, data).and_return(valid_message)
+      expect(Natero::Response).to receive(:new)
+
+      Natero::Event.create(evt, 'test')
+    end
+  end
+
+  describe 'identify_user' do
+    it 'sends the proper request' do
+      evt = event.new(1, 'test')
+      details = 'detail!'
+      request_url = "https://events.natero.com/v1/#{event_auth_key}/#{event_api_key}"
+      data = {:body=>{id:1, name:'test', action:'identifyUser', details: details},
+              :headers=>{'Content-Type': 'application/json'}}
+      method = :post
+
+      expect(HTTParty).to receive(method).with(request_url, data).and_return(valid_message)
+      expect(Natero::Response).to receive(:new)
+
+      Natero::Event.identify_user(evt, details)
+    end
+  end
+
+  describe 'identify_account' do
+    it 'sends the proper request' do
+      evt = event.new(1, 'test')
+      details = 'detail!'
+      request_url = "https://events.natero.com/v1/#{event_auth_key}/#{event_api_key}"
```

Comment:
Could be the base url's in a constant / `let` so that when they update to v2, it's marginally easier to keep specs up-to-date.